src/data/faq.json

```
@@ -214,6 +214,16 @@
                             "<b>Step-by-step instructions:</b>",
                             "To create a printout of your EU COVID Digital Certificate, please follow these steps: <ul><li>Tap on the 'Certificates' tab in the Corona-Warn-App</li><li>Select a person, by tapping on a corresponding certificate in the certificate overview</li><li>Scroll down and select a certificate of this person</li><li>In this detail view, tap on the three-dot menu in the upper right corner (Android) or on 'More' at the bottom (iOS)</li><li>Tap on 'Display print version'</li><li>Tap on 'Continue'</li><li>Tap on the icon for 'Print' or for 'Share' in the upper right corner</li><li>In the following view, you can print the created PDF document using the standard functions of the operating system, import it into other apps or even share it</li></ul>"
                         ]
+                    },
+                    {
+                        "title": "How are certificates verified by third parties?",
+                        "anchor": "eu_dcc_check",
+                        "active": true,
+                        "textblock": [
+                            "In Germany, third parties can only reliably check whether it is a valid vaccination certificate, recovery certificate, or test certificate using the CovPassCheck app. Additionally, an identity check with photo ID should take place. Please note that other countries may use different apps for certificate verification.",
+                            "A pure visual check of the certificates is not sufficient, as manipulated screenshots, apps or system settings (e.g. changed date) could be used, for example. The CovPassCheck-App is a secure solution for e.g. traders and public authorities to reliably check EU digital COVID certificates. More information and download links to the CovPassCheck app can be found here: <a href='https://www.digitaler-impfnachweis-app.de/en/covpasscheck-app/l' target='_blank' rel='noopener noreferrer'>Check EU COVID certificates directly via app</a>. Please note that third parties in other countries may use other apps to verify certificates.",

```

```suggestion
                            "A pure visual check of the certificates is not sufficient, as manipulated screenshots, apps or system settings (e.g. changed date) could be used, for example. The CovPassCheck-App is a secure solution for e.g. traders and public authorities to reliably check EU digital COVID certificates. More information and download links to the CovPassCheck app can be found here: <a href='https://www.digitaler-impfnachweis-app.de/en/covpasscheck-app/' target='_blank' rel='noopener noreferrer'>Check EU COVID certificates directly via app</a>. Please note that third parties in other countries may use other apps to verify certificates.",
```

Error in hyperlink `https://www.digitaler-impfnachweis-app.de/en/covpasscheck-app/l` with extra character at end.

This was caught by check_links.js! 🙂